packages/vitest/src/api/setup.ts

```diff
@@ -121,6 +123,11 @@ class WebSocketReporter implements Reporter {
     if (this.clients.size === 0)
       return
 
+    packs.forEach(([, result]) => {
+      if (result?.error)
```

Comment:
Error can be anything, there should be some kind of check.

Reply:
According to TS it can only be an `ErrorWithDiff` here 👍. Are there cases where at runtime it doesn't actually match this 🙂? If so we probably should update the types 🙂

<img width="385" alt="image" src="https://user-images.githubusercontent.com/4410247/209480339-7623c164-3ab4-420d-9e4b-cf18109f3d3b.png">